examples/http/rabbitmq_nodes.js

```diff
@@ -0,0 +1,317 @@
+/** 
+ * This driver gets node metrics from RabbitMQ management plugin provided an HTTP-based API using HTTP agent
+ * Communication protocol is http
+ * Communicate with http api using USERNAME and PASSWORD
+ * The rabbitmq cluster name is modified in each installation.So, you can find it in the right side of the rabbitmq GUI. example: Cluster rabbit@319ac541e311
```

Comment:
Correct it by:
The rabbitmq cluster name is modified in each installation. it should be found in in the top right corner of the RabbitMQ web application after making the signin. example: Cluster rabbit@319ac541e311